modules/enterprise-management/src/test/java/com/ibm/cloud/platform_services/enterprise_management/v1/EnterpriseManagementIT.java

```diff
@@ -1,433 +1,563 @@
+    assertNotNull(config);
+    assertFalse(config.isEmpty());
+    assertEquals(service.getServiceUrl(), config.get("URL"));
+
+    authType = config.get("AUTHTYPE");
+    assertNotNull(authType);
+
+    authUrl = config.get("AUTH_URL");
+    assertNotNull(authUrl);
+
+    apiKey = config.get("APIKEY");
+    assertNotNull(apiKey);
+
+    enterpriseId = config.get("ENTERPRISE_ID");
+    assertNotNull(enterpriseId);
+
+    accountId = config.get("ACCOUNT_ID");
+    assertNotNull(accountId);
+
+    accountIamId = config.get("ACCOUNT_IAM_ID");
+    assertNotNull(accountIamId);
+
+    System.out.println("Setup complete.");
+  }
+
+  @Test
+  public void testCreateAccountGroup() throws Exception {
+    try {
+      String parent = String.format(createAccountGroupCrn, accountId, enterpriseId);
+      CreateAccountGroupOptions createAccountGroupOptions = new CreateAccountGroupOptions.Builder()
+          .parent(parent)
+          .name(firstExampleAccountGroupName)
+          .primaryContactIamId(accountIamId)
+          .build();
+
+      // Invoke operation
+      Response<CreateAccountGroupResponse> response = service
+          .createAccountGroup(createAccountGroupOptions)
+          .execute();
+      // Validate response
+      assertNotNull(response);
+      assertEquals(response.getStatusCode(), 201);
+
+      CreateAccountGroupResponse createAccountGroupResponseResult = response.getResult();
+      assertNotNull(createAccountGroupResponseResult);
+
+      firstExampleAccountGroupId = createAccountGroupResponseResult.getAccountGroupId();
+    } catch (ServiceResponseException e) {
+      fail(String.format("Service returned status code %d: %s%nError details: %s", e.getStatusCode(), e.getMessage(),
+          e.getDebuggingInfo()));
+    }
+  }
+
+  @Test
+  public void testCreateAnotherAccountGroup() throws Exception {
+    try {
+      String parent = String.format(createAccountGroupCrn, accountId, enterpriseId);
+      CreateAccountGroupOptions createAccountGroupOptions = new CreateAccountGroupOptions.Builder()
+          .parent(parent)
+          .name(secondExampleAccountGroupName)
+          .primaryContactIamId(accountIamId)
+          .build();
+
+      // Invoke operation
+      Response<CreateAccountGroupResponse> response = service
+          .createAccountGroup(createAccountGroupOptions)
+          .execute();
+      // Validate response
+      assertNotNull(response);
+      assertEquals(response.getStatusCode(), 201);
+
+      CreateAccountGroupResponse createAccountGroupResponseResult = response.getResult();
+      assertNotNull(createAccountGroupResponseResult);
+
+      secondExampleAccountGroupId = createAccountGroupResponseResult.getAccountGroupId();
+    } catch (ServiceResponseException e) {
+      fail(String.format("Service returned status code %d: %s%nError details: %s", e.getStatusCode(), e.getMessage(),
+          e.getDebuggingInfo()));
+    }
+  }
+
+  @Test
```

Comment:
```suggestion
  @Test```suggestion
  @Test(dependsOnMethods= {"testCreateAccountGroup", "testCreateAnotherAccountGroup"})
```
By using the "dependsOnMethods" attribute of the Test annotation, we can define dependencies between the test methods which ensures that they are run in the correct order, and also that if a dependent method fails, the test method that depends on it won't be run.   In this case, it makes sense to have the list operation depend on the two create operations.
```